src/cljs/hakukohderyhmapalvelu/components/common/button.cljs

```diff
@@ -36,18 +36,49 @@
                           [:focus (merge hover-styles
                                          {:filter effects/drop-shadow-effect-blue})]]}))
 
+(defn- make-text-button-styles [style-prefix]
+  {:background      "none"
+   :border          "none"
+   :color           colors/blue
+   :cursor          "pointer"
+   :font-size       "1rem"
+   :grid-area       style-prefix
+   :margin          0
+   :padding         0
+   :text-decoration "underline"
+   ::stylefy/mode   [[":hover:not(:disabled)" {:color colors/blue-lighten-1}]]})
```

Comment:
oisko hoveriin vielä cursor: pointer muutos :)

Reply:
vai tuleeko se buttonista out of the box 🤔

Reply:
Perii sen tuosta buttonin tyylistä `:cursor          "pointer"`